test/test.js

```diff
@@ -259,15 +259,28 @@ QUnit.module('@yelm', () => {
 		assert.equal(divComponent.innerText, 'test New Prop Value');
 	});
 
-	QUnit.test('ObserveArray.map() shouldn\'t convert jsx to proxies', (assert) => {
-		const myList = new ObserveArray([ 1 ]);
+	QUnit.test('should not make jsx observable', (assert) => {
```

Comment:
I find this test title a bit misleading, I think it should be something like:
`should not convert React elements into observable objects and arrays' or something similar.

React elements are the things returned by the function calls  that JSX transpile down to.